### PR TITLE
fixed doc param inside Query Builder

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**
Fixed a parameter documentation from string to mixed because it can be a string or an array

### CODE REVIEW

File: lib/Doctrine/DBAL/Query/QueryBuilder.php [hunk 1]
```diff
@@ -409,7 +409,7 @@ public function getMaxResults()
      * 'groupBy', 'having' and 'orderBy'.
      *
      * @param string  $sqlPartName
-     * @param string  $sqlPart
+     * @param mixed  $sqlPart
```
Comment (Ocramius) [inline lib/Doctrine/DBAL/Query/QueryBuilder.php]: Should probably be `string`, no?

Comment (AlessandroMinoccheri) [inline lib/Doctrine/DBAL/Query/QueryBuilder.php]: In some part of the code for example in line 530 is passed as an array not as a string right?

Comment (Ocramius) [inline lib/Doctrine/DBAL/Query/QueryBuilder.php]: That's because it's upcast (line 423)

Comment (AlessandroMinoccheri) [inline lib/Doctrine/DBAL/Query/QueryBuilder.php]: Yes I have seen but I thought that you can call this function with a string or with an array so I thought that mixed type is better than string inside documentation.
If It's better `string` I understand but there are two ways for me to call this function for that parameter

Comment (Ocramius) [inline lib/Doctrine/DBAL/Query/QueryBuilder.php]: @AlessandroMinoccheri use `string|string[]` or whatever matches then :-)

**Comment by morozov** [comment]
Obsolete as of https://github.com/doctrine/dbal/pull/3836.

**Pull request closed without merging**